packages/collector/test/tracing/database/mssql/test.js

```diff
@@ -20,9 +20,14 @@ describe('tracing/mssql', function () {
 
     // The latest version of mssql (v11) no longer supports Node.js versions 16 and below.
     // For more details, see the release notes: https://github.com/tediousjs/node-mssql/releases/tag/v11.0.0
+    // In the next major release we can discontinue testing with multiple versions. Currently, we maintain this
+    // testing as the latest version of mssql does not support Node.js versions 14 and 16.
     if (mssqlVersion === 'latest') {
       mochaSuiteFn = semver.lt(process.versions.node, '18.0.0') ? describe.skip : mochaSuiteFn;
     }
+    if (mssqlVersion === 'v10') {
+      mochaSuiteFn = semver.lt(process.versions.node, '16.0.0') ? describe.skip : mochaSuiteFn;
```

Comment:
I dont understand the fix. There is no context provided. Why do we have to skip v10 suddenly?

See 
https://github.com/tediousjs/node-mssql/blob/v10.0.4/package.json#L49